network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -175,9 +176,7 @@ class ParticipatePage2(PrimaryPage):
     )
 
     ctaHeroSubhead = RichTextField(
-        features=[
-            'bold', 'italic', 'link',
-        ],
+        features=(base_rich_text_options),
```

Comment:
Putting `()` around things is a dangerous thing in Python - at best it does nothing, but often it completely changes the data type, so let's not add them here =)

Reply:
Added to latest commit! Marking as resolved